Tests/StencilTests/EnvironmentSpec.swift

```diff
@@ -32,9 +39,304 @@ func testEnvironment() {
 
       try expect(result) == "here"
     }
+
+    func expectedSyntaxError(token: String, template: Template, description: String) -> TemplateSyntaxError {
+      let range = template.templateString.range(of: token)!
+      let rangeLine = template.templateString.rangeLine(range)
+      let sourceMap = SourceMap(filename: template.name, line: rangeLine)
+      let token = Token.block(value: token, at: sourceMap)
+      return TemplateSyntaxError(reason: description, token: token, stackTrace: [])
+    }
+
+    func expectError(reason: String, token: String,
+                     file: String = #file, line: Int = #line, function: String = #function) throws {
+      let expectedError = expectedSyntaxError(token: token, template: template, description: reason)
+      
+      let error = try expect(environment.render(template: template, context: ["names": ["Bob", "Alice"], "name": "Bob"]),
```

Comment:
Both the declaration of this `func` as well as this line have some bizarre indentation. This also applies to the similar functions below.

Reply:
This function is repeated quite a few times. That could be rewritten?

Reply:
I'd prefer this to 6 arguments stacked on each other
this can be cleared up sometime later, this PR is big without it